src/Core/Commander/ManagerCommands.js

```diff
@@ -127,7 +133,16 @@ define('Core/Commander/ManagerCommands', [
             while (this.queueAsync.length > 0) {
                 var cmd = this.queueAsync.peek();
 
-                if (cmd.earlyDropFunction && cmd.earlyDropFunction(cmd)) {
+                var requester = cmd.requester;
+                if (requester.disposed) {
+                    this.queueAsync.dequeue();
+                } else if (cmd.earlyDropFunction && cmd.earlyDropFunction(cmd)) {
+                    while (requester.children.length > 0) {
+                        var child = requester.children[0];
+                        child.dispose();
+                        requester.remove(child);
+                    }
+                    requester.pendingSubdivision = false;
```

Comment:
It's not exactly restored. It is part of this commit a1135483f65c3c07f0568434241697eabbe985c3 that fixed some commands being launched for disposed nodes.
With the addition of the early drop function, the bug seems to have disappeared, I'll remove the commit.